__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require('supertest')
+const db  = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const data = require('../db/data/test-data/index')
+const express = require('express')
+const app = require('../app')
+
+beforeAll(()=>seed(data))
+afterAll(()=>db.end())
+
+describe('api/topics',()=>{
+    test('GET:200 should return an array of topic objects with the properties slug and description',()=>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body})=>{
+            expect(body.length).toBe(3)
+            body.forEach((row)=>{
+                expect(row).toEqual(
+                    expect.objectContaining({
+                        slug: expect.any(String),
+                        description: expect.any(String)
+                    })
+                )
+            })
+        })
+
+    })
+    test('GET:404 incorrect endpoint should return an object with a key of msg and value of Bad request',()=>{
```

Comment:
this is a general 404 and not specific to the topics endpoint - this would do the same for any non-existent endpoint e.g. /api/bananas so doesnt need to be associated with the topics